arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/GeoAggregateRESTService.java

```diff
@@ -72,74 +77,83 @@ public GeoAggregateRESTService(ExploreService exploreService) {
     @GET
     @Produces(UTF8JSON)
     @Consumes(UTF8JSON)
-    @ApiOperation(value = "GeoAggregate", produces = UTF8JSON, notes = Documentation.GEOAGGREGATION_OPERATION, consumes = UTF8JSON, response = FeatureCollection.class)
-    @ApiResponses(value = {@ApiResponse(code = 200, message = "Successful operation", response = FeatureCollection.class, responseContainer = "FeatureCollection"),
-            @ApiResponse(code = 500, message = "Arlas Server Error.", response = Error.class), @ApiResponse(code = 400, message = "Bad request.", response = Error.class),
-            @ApiResponse(code = 501, message = "Not implemented functionality.", response = Error.class)})
+    @Operation(summary = "GeoAggregate",  description = Documentation.GEOAGGREGATION_OPERATION)
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(array = @ArraySchema(schema = @Schema(implementation = FeatureCollection.class)))),
```

Comment:
It 's not an array, the object feature collection contains an attribute features that is a list

Reply:
it applies for all geo-endpoints

Reply:
Done, please review that I did not forget any.